trestle/core/commands/assemble.py

```diff
@@ -15,10 +15,189 @@
 # limitations under the License.
 """Trestle Assemble Command."""
 
+import argparse
+import logging
+from pathlib import Path
+from typing import Type, TypeVar
+
 from ilcli import Command  # type: ignore
 
+from trestle.core import const
+from trestle.core.models.actions import CreatePathAction, WriteFileAction
+from trestle.core.models.elements import Element
+from trestle.core.models.file_content_type import FileContentType
+from trestle.core.models.plans import Plan
+from trestle.oscal import assessment_plan
+from trestle.oscal import assessment_results
+from trestle.oscal import catalog
+from trestle.oscal import component
+from trestle.oscal import poam
+from trestle.oscal import profile
+from trestle.oscal import ssp
+from trestle.oscal import target
```

Comment:
Giveen there is a common pattern of importing everything we should find a simpler way to do this (it should be viable in one line).

Reply:
agreed. We now have 3 commands that do this.